include/functions.h

```diff
@@ -990,8 +990,8 @@ void Color_RGBA8_Copy(Color_RGBA8* dst, Color_RGBA8* src);
 void func_80078884(u16 sfxId);
 void func_800788CC(u16 sfxId);
 void func_80078914(Vec3f* arg0, u16 sfxId);
-void Health_InitData(GlobalContext* globalCtx);
-void Health_UpdateData(GlobalContext* globalCtx);
+void Health_Init(GlobalContext* globalCtx);
+void Health_Update(GlobalContext* globalCtx);
```

Comment:
we may want to consider calling this HealthMeter or something, since it really is just the graphical display